src/pages/Main/components/OfferCard.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import './OfferCard.scss';
+
+export default class OfferCard extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: this.props.data,
```

Comment:
constructor 안에서 props에 접근할 때는 this가 없어도 됩니다 !

```suggestion
      data: props.data,
```

Reply:
앗! 감사합니다